IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -228,16 +230,19 @@ block ReaderTMY3 "Reader for TMY3 weather data"
 protected
   final parameter String absFilNam = IBPSA.BoundaryConditions.WeatherData.BaseClasses.getAbsolutePath(filNam)
     "Absolute path of the file";
+  // For evalating time span of weather data
+  parameter Modelica.SIunits.Time[2] timeSpan(start = {0.0, 3600.0}) = IBPSA.BoundaryConditions.WeatherData.BaseClasses.getTimeSpanWeatherData(absFilNam, tableName)  "start and end time of weather data";
```

Comment:
Avoid too long lines as they don't display well in some editors

Reply:
Ok. Have spread some lines over several rows, hope they are better now.